system/autoware_error_monitor/config/diagnostic_aggregator/sensing.yaml

```diff
@@ -18,7 +18,7 @@
 
         lidar:
           type: 'diagnostic_aggregator/AnalyzerGroup'
-          path: velodyne
+          path: lidar
```

Comment:
You should add an AnalyzerGroup.
We need `/sensing/lidar/velodyne/*` and `/sensing/lidar/livox/*`

Reply:
Thanks. Fixed at  261fc14